docs/synapse/userguides/userguide_section11.rst

```diff
@@ -0,0 +1,128 @@
+
+``cli> ask inet:ipv4=1.2.3.4``
+
+Note that Storm accepts the IP address in its “intuitive” form (dotted decimal notation), even though Synapse stores IP addresses as integers (IP ``1.2.3.4`` is stored as integer ``16909060``). The analyst does not need to convert the IP to integer form to run the query, nor do they need to escape the IP with quotes (``“1.2.3.4”``) to indicate it is a string representation of the data. (Generally speaking, double quotes only need to be used when input contains characters that would otherwise be interpreted as "end of data" (space, comma) or other specialized input (e.g, escape characters) by the Synapse parser.)
+
+In addition, the Storm syntax should feel intuitive to the user, like asking a question: “Tell me about (ask about) the IP address 1.2.3.4”. In reality, Storm translates that simplified syntax into the correct syntax for the underlying ``lift()`` operator, which actually retrieves the data:
+
+``cli> ask inet:ipv4=1.2.3.4``
+
+is equivalent to:
+
+``cli> ask lift(inet:ipv4,1.2.3.4)``
+
+Analysts still need to learn the Storm “language” and master enough command-line syntax to perform tasks and find help when necessary. However, the intent is for Storm to function more like “how do I ask this question of the data?” and not “how do I write a program to get the data I need?”
+
+Finally – and most importantly – giving analysts direct access to Storm to allow them to create arbitrary queries provides them with an extraordinarily powerful analytical tool. Analysts are not constrained by working with a set of predefined queries provided to them through a GUI or an API. Instead, they can follow their analysis wherever it takes them, creating queries as needed and working with the data in whatever manner is most appropriate to their research.
+
+Storm Operators
+---------------
+
+Storm is based on an underlying set of **operators** that allow interaction with a Synapse Cortex and its data. All Storm operators can be accessed from the Synapse command line interface via the ``ask`` command, using either operator syntax or the shortened macro syntax where available. Most operators (other than ``lift()`` and a few others used to select or retrieve data) require an existing data set on which to operate. This data set is typically the output of a previous Storm query whose results are the nodes you want to modify or otherwise work with. Operators may also require one or more parameters to further specify what you want to do with the result set.
+
+Storm queries can be as simple as asking to lift a single node:
+
+``cli> ask inet:fqdn=woot.com``
+
+Alternately, they can be highly complex, chaining a series of operators to lift a set of nodes and perform a series of additional filter and pivot operations that follow a line of analysis across the data:
+
+``cli> ask inet:fqdn*tag=tc.t12 -> inet:dns:a:fqdn inet:dns:a:ipv4 -> inet:ipv4 -#anon.tor -#anon.vpn -> inet:dns:a:ipv4 inet:dns:a:fqdn -> inet:fqdn``
```

Comment:
This line goes off the page for me, but I am not sure how to fix it. It might be possible to use "code blocks": https://rest-sphinx-memo.readthedocs.io/en/latest/ReST.html#code-blocks